tables/tests/test_attributes.py

```diff
@@ -1,7 +1,7 @@
 """This test unit checks node attributes that are persistent (AttributeSet)."""
 
+import datetime as dt
```

Comment:
It is my personal taste but I would prefer to keep the original name in this case

Reply:
No problem, changed.